ml-search-server/api/controllers/searchController.js

```diff
@@ -1,37 +1,39 @@
-    res.send({
-        totalCount: 100,
-        data: [{
-                url: '/D/POC/ml-search-poc/1.json',
-                text: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.'
-            },
-            {
-                url: '/D/POC/ml-search-poc/2.json',
-                text: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.'
-            },
-            {
-                url: '/D/POC/ml-search-poc/3.json',
-                text: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.'
-            },
-            {
-                url: '/D/POC/ml-search-poc/4.json',
-                text: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.'
-            },
-            {
-                url: '/D/POC/ml-search-poc/5.json',
-                text: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.'
-            },
-            {
-                url: '/D/POC/ml-search-poc/6.json',
-                text: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.'
-            },
-            {
-                url: '/D/POC/ml-search-poc/7.json',
-                text: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.'
-            }
-        ]
+    var qb = marklogic.queryBuilder;
+    const pageNumber = req.query.page_no;
+    //const pageSize = 10;
+    const pageSize = req.query.page_size;
+    const start = (pageNumber - 1) * pageSize;
+    const query = db.documents.query(
+        qb.where(qb.byExample({ tags: req.query.text }))
+    )
+    query.result(function(documents) {
+
+        const end = start + pageSize;
+
+        res.header("Access-Control-Allow-Origin", "*");
```

Comment:
Move this to server.js and it should be allowed for all request from the local machine for now.